etc/doc/tutorial/pl/01.1-Live-Coding.md

```diff
@@ -118,9 +117,9 @@ live_loop :boom do
 end
 ```
 
-Baw się i eksperymentuj tym kawałkiem kodu aż twoja ciekwość sprawi, że 
-zaczniesz zastanawiać się jak to wszystko właściwie działa oraz co innego 
-można jeszcze wyczarować za pomocą tego narzędzia. Jeśli ten moment już 
-nastąpił to jesteś gotów aby przeczytać ten samouczek do końca.
+Baw się i eksperymentuj tym kawałkiem kodu do momentu gdy twoja ciekwość 
```

Comment:
@nicoder yup You're Right :) I'll fix it this evening or feel free to create Your own PR :)